console/src/main/java/org/nervos/ckb/example/TransactionExample.java

```diff
@@ -0,0 +1,145 @@
+
+  private Script inputLockScript;
+  private String privateKeyHex;
+  private CKBService ckbService;
+
+  public TransactionExample(String privateKeyHex, Script inputLockScript) {
+    this.privateKeyHex = privateKeyHex;
+    this.inputLockScript = inputLockScript;
+    ckbService = CKBService.build(new HttpService(NODE_URL));
+  }
+
+  public String sendCapacity(List<Receiver> receiverList) throws Exception {
+    Transaction transaction = generateTx(receiverList);
+    CkbTransactionHash temp = ckbService.sendTransaction(transaction).send();
+    if (temp.result == null) {
+      System.out.println(temp.error.message);
+    }
+    return temp.getTransactionHash();
+  }
+
+  private Transaction generateTx(List<Receiver> receiverList) throws Exception {
+    BigInteger needCapacities = BigInteger.ZERO;
+    for (Receiver receiver : receiverList) {
+      BigDecimal bigDecimal = new BigDecimal(receiver.capacity);
+      needCapacities = needCapacities.add(bigDecimal.toBigInteger());
+    }
+    if (needCapacities.compareTo(new BigDecimal(MIN_CAPACITY).toBigInteger()) < 0) {
+      throw new Exception("Less than min capacity");
+    }
+
+    CkbSystemContract systemContract =
+        SystemContract.getSystemContract(ckbService, Network.TESTNET);
+    CellInputsAndBalanceSum cellInputsAndBalanceSum =
+        generateInputs(inputLockScript.scriptHash(), needCapacities);
+    if (cellInputsAndBalanceSum.capacity.compareTo(needCapacities) < 0) {
+      throw new Exception("No enough Capacities");
+    }
+    List<CellOutput> cellOutputs = new ArrayList<>();
+    receiverList.forEach(
+        (receiver -> {
+          AddressUtils addressUtils = new AddressUtils(Network.TESTNET);
+          String blake2b = addressUtils.getBlake160FromAddress(receiver.address);
+          cellOutputs.add(
+              new CellOutput(
+                  receiver.capacity,
+                  "0x",
+                  new Script(systemContract.systemScriptCellHash, Arrays.asList(blake2b))));
+        }));
+    if (cellInputsAndBalanceSum.capacity.compareTo(needCapacities) > 0) {
+      cellOutputs.add(
+          new CellOutput(
+              cellInputsAndBalanceSum.capacity.subtract(needCapacities).toString(10),
+              "0x",
+              inputLockScript));
+    }
+    Transaction transaction =
+        new Transaction(
+            "0",
+            Arrays.asList(new OutPoint(null, systemContract.systemScriptOutPoint)),
+            cellInputsAndBalanceSum.inputs,
+            cellOutputs,
+            new ArrayList<>());
+    String txHash = ckbService.computeTransactionHash(transaction).send().getTransactionHash();
+    Witness witness = new Witness(Numeric.toBigInt(privateKeyHex), txHash);
+    cellInputsAndBalanceSum.inputs.forEach(
+        cellInput -> {
+          transaction.witnesses.add(witness);
+        });
+    return transaction;
+  }
+
+  private CellInputsAndBalanceSum generateInputs(String lockHash, BigInteger needCapacities)
```

Comment:
If you want to describe all information, method name should be generateCellInputsAndBalanceSum